packages/client/src/index.ts

```diff
@@ -9,11 +10,14 @@ import { DataChannelDebouncer } from "../../../plugins/webrtc/src/datachannel-de
 import type { IOSocket } from '../../../server/src/io';
 import { MediaObject } from '../../../server/src/plugin/mediaobject';
 import { attachPluginRemote } from '../../../server/src/plugin/plugin-remote';
+import { ClusterObject, ConnectRPCObject } from '../../../server/src/plugin/plugin-remote-worker';
```

Comment:
This import pulls in a whole bunch of dependencies. You must use `import type` to prevent that. Ideally, these types maybe ought to live in their own .ts file.